.github/PULL_REQUEST_TEMPLATE/documentation_template.md

```diff
@@ -0,0 +1,76 @@
+# Documentatioon 
+
+**Title:** 
+
+## Category
+
+Which category does this documentation add/updated/remove?
+- [ ] code documentation
+- [ ] wiki
+- [ ] README
+- [ ] installation guide
+    - [ ] OS specifics
+    - [ ] dependency
+    - [ ] environment 
+- [ ] tutorials 
+    - [ ] plotting
+    - [ ] io
+    - [ ] utils/exceptions
+- [ ] examples
+- [ ] citing/licensing
+- [ ] trouble shooting
+    - [ ] installation
+    - [ ] tutorials
+- [ ] history
+- [ ] guidelines 
+    - [ ] developer
+    - [ ] communication
+
+### Details
+
+*Please provide more details on the add/updated/removed documentation on the selected categories.* 
+
+### Goals
+What goals are you trying to achieve with the add/updated/removed documentation? 
+- [ ] readability (fixing typos/grammar)
+- [ ] conciseness (making it less confusing)
+- [ ] updating (updating old plots/code examples)
+- [ ] warnings (warning of caveats or potential issues)
+- [ ] user-interface (making something more user friendly)
+
+## Issue reference
+
+**Issue:** *issue number*
+
+## Checklist
+
+- [ ] spell checker
```

Comment:
Would be helpful to indicate here whether we are using US or UK spelling standards in the documentation.

Reply:
I have no strong opinion on either or personally and wouldn't want the person to be bogged down by this, but I will recommend the US. People will probably use default spell checkers, so I wouldn't be surprised if we still have some UK spelling in there or typos.